### ISSUE TITLE
Loading Indication

### ISSUE THREAD

**Issue by nixpulvis** [kisonecat/tikzjax#4]
Generating graphs client side is awesome, but can be rather slow. Ideally there'd be some kind of indication that the contents of a `\begin{tikzpicture}` is being rendered.

For example, I've copied the code for a somewhat large complete graph provided [here](http://www.texample.net/tikz/examples/complete-graph/). Note that the code in the example proper doesn't work with this library yet (something about `mycount`), but the simplified example in the comments does.

**Comment by kisonecat** [comment]
Yes there really should be a loading indicator ... some sort of spinning placeholder so the user knows to expect a figure.

(There's certainly a lot of room for improving this.  I'll be working on some of the tex-in-a-browser stuff over winter break.)

**Comment by StevenClontz** [comment]
I just thought to make this request in anticipation for using this library in PreTeXt, so +1
